src/main/java/ymfc/parser/Parser.java

```diff
@@ -301,6 +301,16 @@ private static EditCommand getEditCommand(String args) throws InvalidArgumentExc
                 .filter(s -> !s.isEmpty())
                 .collect(Collectors.toCollection(ArrayList::new));
 
+        //@@author gskang
+        // Extract step identifiers (s1, s2, ...) and validate for duplicates or missing numbers
+        List<String> stepIdentifiers = Arrays.stream(stepString.split("\\s+"))
+                .filter(step -> step.matches("[sS][0-9]+/.*")) // Ensure the string matches the step format
+                .map(step -> step.split("/")[0]) // Extracts "s1", "s2", etc.
+                .toList();
+
+        validateStepNumbers(stepIdentifiers); // Check for missing/duplicate numbers
+        //@@author
+
```

Comment:
Is the line being attributed to gskang because of something that happened in an earlier PR? This lines look new to me, I would have thought you wrote them

Reply:
Yeah I copy-pasted the `validateStepNumbers` part from `addCommand`, which was created by @gskang-22, so I have to use the @@author tag to make it recognizable by RepoSense